src/game_inventory.cpp

```diff
@@ -565,13 +606,26 @@ class comestible_inventory_preset : public inventory_selector_preset
             }, _( "QUENCH" ) );
 
             append_cell( [&you]( const item_location & loc ) {
-                const item &it = *loc;
-                if( it.has_flag( flag_MUSHY ) ) {
-                    return highlight_good_bad_none( you.fun_for( *loc ).first );
+                int joy = you.fun_for( *loc ).first;
+                std::string joy_str;
+                if( loc->has_flag( flag_MUSHY ) ) {
+                    joy_str = highlight_good_bad_none( joy );
                 } else {
-                    return good_bad_none( you.fun_for( *loc ).first );
+                    joy_str = good_bad_none( joy );
                 }
-            }, _( "JOY" ) );
+
+                int max_joy = you.fun_for( *loc, true ).first;
+                // max_joy should be 3 wide for aligment of '/'
+                if( max_joy == 0 ) {
+                    // this rarely will not be empty string
+                    return joy_str;
+                } else if( max_joy == joy ) {
+                    return joy_str + std::string( joy > 0 ? "<good>/  =</good>" : "<bad>/  =</bad>" );
+                } else {
+                    // max_joy has a sign, so up to two spaces have to be right_justified for max_joy to be 3 wide
+                    return joy_str + "/" + std::string( 2 - n_digits( max_joy ), ' ' ) + good_bad_none( max_joy );
```

Comment:
Rather than adding `n_digits` you can [use printf format specifiers](https://godbolt.org/#g:!((g:!((g:!((h:codeEditor,i:(filename:'1',fontScale:14,fontUsePx:'0',j:1,lang:c%2B%2B,selection:(endColumn:26,endLineNumber:5,positionColumn:26,positionLineNumber:5,selectionStartColumn:26,selectionStartLineNumber:5,startColumn:26,startLineNumber:5),source:'%23include+%3Cstdio.h%3E%0A%0Aint+main()%0A%7B%0A++++printf(%22X%253sX%22,+%22Y%22)%3B%0A%7D'),l:'5',n:'0',o:'C%2B%2B+source+%231',t:'0')),k:50,l:'4',m:100,n:'0',o:'',s:0,t:'0'),(g:!((g:!((h:compiler,i:(compiler:g102,filters:(b:'0',binary:'1',commentOnly:'0',demangle:'0',directives:'0',execute:'0',intel:'0',libraryCode:'0',trim:'1'),flagsViewOpen:'1',fontScale:14,fontUsePx:'0',j:1,lang:c%2B%2B,libs:!(),options:'-O3',selection:(endColumn:1,endLineNumber:1,positionColumn:1,positionLineNumber:1,selectionStartColumn:1,selectionStartLineNumber:1,startColumn:1,startLineNumber:1),source:1,tree:'1'),l:'5',n:'0',o:'x86-64+gcc+10.2+(C%2B%2B,+Editor+%231,+Compiler+%231)',t:'0')),k:50,l:'4',m:50,n:'0',o:'',s:0,t:'0'),(g:!((h:output,i:(compiler:1,editor:1,fontScale:14,fontUsePx:'0',wrap:'1'),l:'5',n:'0',o:'Output+of+x86-64+gcc+10.2+(Compiler+%231)',t:'0')),header:(),l:'4',m:50,n:'0',o:'',s:0,t:'0')),k:50,l:'3',n:'0',o:'',t:'0')),l:'2',n:'0',o:'',t:'0')),version:4) for this.  Something like
```suggestion
                    return string_format( "%s/%3s", joy_str, good_bad_none( max_joy ) );
```

Reply:
That's cool! I somehow forgot about these...

Reply:
It was a tad bit more complicated, as `good_bad_none( max_joy ) )` for joy=5 returns `"<good>5</good>"`, so using function `good_bad_none` is not great anymore.

Alternatively, I could add optional parameter `int width`, default to 0, then the code would become:
`return string_format( "%s/%s", joy_str, good_bad_none( max_joy, 3 ) )`
But this is the only place that needs it, so I decided against it.